.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Android-ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: docker://fabernovel/android:api-31-v1.6.0
+    steps:
+      - uses: actions/checkout@v4
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```

Comment:
Is this needed? Jenkins CI does not need this.

Reply:
Removed @jlin-sift